SonarQube.Client/Api/V2_10/GetPluginsRequest.cs

```diff
@@ -38,7 +38,7 @@ public class GetPluginsRequest : RequestBase<SonarQubePlugin[]>, IGetPluginsRequ
         private SonarQubePlugin ToPlugin(PluginResponse response) =>
             new SonarQubePlugin(response.Key, response.Version);
 
-        private class PluginResponse
+        private sealed class PluginResponse
```

Comment:
Most of the changes are just to mark the private data classes as `sealed`.
I'll explicitly comment the two or three that are not.